### DISCUSSION TITLE
Recreate page listing software services/deployments

### DISCUSSION THREAD

**Discussion by bzg** [codegouvfr/sill-web#464] [Ideas]
La version précédente du SILL permettait de lister tous les services déployés à partir d'un logiciel libre du SILL.

Cette version pourrait recréer une telle page, sur le modèle de la page du catalogue de logiciels.

**Comment by bzg** [comment]
Ce besoin est propre à la mission logiciels libres : d'autres instances d'un « catalogue de logiciels » pourraient ne pas avoir besoin de lister les « instances » de ces logiciels.

Voir comment implémenter ça de façon à ce que cela soit optionnel et sans créer trop de dette ?

**Comment by lgrd** [comment]
@bzg J'ai du mal à voir si ce dont tu parles là n'est pas l'onglet "Instances" du Catalogi actuel. Any insights ? :)

**Reply by bzg** [reply]
De mémoire, l'onglet n'existait pas en juillet 2023 : l'onglet "Instances" répond désormais au besoin.

La dernière question portait ensuite sur Catalogi: est-ce que d'autres instances de Catalogi auront besoin de cette fonctionnalité?

**Comment by lgrd** [comment]
Merci ! :) 
Il me semble que l'onglet n'apparaît que s'il y en a d'enregistré. Et je ne serais pas étonné que l'onglet soit totalement désactivable.

**Comment by bzg** [comment]
D'ac, j'ai créé [cette issue](https://github.com/codegouvfr/catalogi/issues/474) pour garder trace de la demande de fonctionnalité sur le fait d'avoir l'onglet en option.  Il y a plus que l'onglet : il y a aussi du wording ("Ajouter une instance") et peut-être d'autres aspects dans le code.
